package/base-files/files/etc/rc.common

```diff
@@ -62,20 +62,36 @@ depends() {
 	return 0
 }
 
+EXTRA_HELP=""
+EXTRA_COMMANDS=""
+extra_command() {
+	local cmd="$1"
+	local help="$2"
+
+	local extra
+
+	extra="$(printf "%-16s%s" "${cmd}" "${help}")"
+
+	EXTRA_HELP="\t${extra}\n${EXTRA_HELP}"
```

Comment:
Doesn't this reverse the order as existing entries are added below?

Reply:
Yes your right. But it looks that way now too. So that`s ok.

Reply:
Are you sure? Looking at [the curernt] rc.common it seems it would be `start[...]\n stop[...]`. With your implementation it would be stop followed by start.

Reply:
That's right! For the standard targets of rc.common I have to change that. Thanks for the hint.

Reply:
I would do it the other way around, else it look counter-intuitive to me...

Reply:
You are mixing two purposes of function to add commands. One is to append commands to the end (normal usage), one is to prepend commands to appear at the beginning of help text (usage in `help()` function). The current implementation always prepends the command to the beginning of help text, which is counter-intuitive and looks weird. Maybe you need two functions?